pages/signup.tsx

```diff
@@ -50,7 +51,8 @@ const SignUp = () => {
       draggable: true,
       progress: undefined,
       theme: "light",
-      });
+    });
+    await sendWelcomeMail(user.email);
```

Comment:
there is no user variable in this file

Reply:
I just checked that user is declared at line 15.

const [user, setUser] = useState({    // Code at line 15